google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingHandler.java

```diff
@@ -35,6 +33,7 @@
 import java.util.logging.LogRecord;
 import java.util.logging.Logger;
 import java.util.logging.SimpleFormatter;
+import com.google.cloud.logging.Logging.WriteOption;
```

Comment:
Yeah let's revert the comment changes.

BTW in IntelliJ when you select code and use the formatter, it will only format that selection.